UHResidentInformationAPI/V1/Infrastructure/Address.cs

```diff
@@ -0,0 +1,26 @@
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace UHResidentInformationAPI.V1.Infrastructure
+{
+    [Table("property")]
+    public class Address
+    {
+       [Column("prop_ref")]
+        [MaxLength(9)]
+        [Key]
+        public int propertyRef { get; set; }
+
+        [Column("house_ref")]
+        [MaxLength(9)]
+        public int houseRef { get; set; } 
+
+         [Column("address1")]
```

Comment:
There's also a `short_address` in this table. 
Would you happen to know the difference?